chips/nrf51/src/trng.rs

```diff
@@ -0,0 +1,132 @@
+//! TRNG driver for the nrf51dk
```

Comment:
For good measure, please add some module level documentation for what this controller is, where to find the documentation for it and any high level quirks (like the fact that it only gives out a single byte at a time).